Makefile

```diff
@@ -0,0 +1,143 @@
+PWD:=$(shell pwd)/
+
+AFL_DIR:=$(PWD)AFL/
+LLVM_MODE_DIR:=$(AFL_DIR)llvm_mode/
+AFL_SHOW_MAP:=$(AFL_DIR)afl-showmap
+AFL_CLANG_FAST:=$(AFL_DIR)afl-clang-fast
+AFL_FUZZ:=$(AFL_DIR)afl-fuzz
+AFL_FLAGS:=-m 128
+
+DEV_KIT_URL:=https://github.com/frida/frida/releases/download/12.11.10/frida-gum-devkit-12.11.10-linux-x86_64.tar.xz
```

Comment:
We support Android and macOS as well (maybe Windows, but I haven't tested), so this should retrieve the devkit depending on the platform. See here: https://github.com/oleavr/ios-inject-custom/blob/master/Makefile#L62